projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -599,4 +626,44 @@ public function create_publicize_connection( $request ) {
 			__( 'Something went wrong while creating a connection.', 'jetpack-publicize-pkg' )
 		);
 	}
+
+	/**
+	 * Update the post with information about shares.
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 */
+	public function update_post_shares( $request ) {
+		$request_body = $request->get_json_params();
+
+		$post_id   = $request->get_param( 'id' );
+		$post_meta = $request_body['meta'];
```

Comment:
Does $request_body always has the `meta` prop? Just asking because if `$post_meta` will be null, the if below might complain where we select `$post_meta[ self::SOCIAL_SHARES_POST_META_KEY ]`

Reply:
Since `meta` is a required property above, it's guaranteed to be present.